server/isomorphic-routing/index.js

```diff
@@ -6,22 +6,32 @@ import { setSection as setSectionMiddlewareFactory } from '../../client/controll
 
 export function serverRouter( expressApp, setUpRoute, section ) {
 	return function( route, ...middlewares ) {
-		expressApp.get(
-			route,
-			setUpRoute,
-			combineMiddlewares(
-				setSectionMiddlewareFactory( section ),
-				...middlewares
-			),
-			serverRender
-		);
+		if ( middlewares.length === 0 && typeof route === 'function' && route.length === 3 ) {
```

Comment:
Yeah, seems you're right.

Then my other excuse is that it's true for `page.js` routes (`page( someMw )` is equivalent to `page( '*', someMw )` , and we want to retain compat with those for isomorphism...